bezier/src/main/java/io/channel/bezier/compose/component/floating_button/FloatingButton.kt

```diff
@@ -0,0 +1,281 @@
+package io.channel.bezier.compose.component.floating_button
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.channel.bezier.BezierIcons
+import io.channel.bezier.BezierTheme
+import io.channel.bezier.compose.R
+import io.channel.bezier.compose.component.avatar.BezierAvatar
+import io.channel.bezier.compose.component.floating_button.properties.BezierFloatingButtonColor
+import io.channel.bezier.compose.component.floating_button.properties.BezierFloatingButtonContent
+import io.channel.bezier.compose.component.floating_button.properties.BezierFloatingButtonSize
+import io.channel.bezier.compose.component.floating_button.properties.BezierFloatingButtonVariant
+import io.channel.bezier.compose.component.loader.BezierLoader
+import io.channel.bezier.compose.component.loader.properties.BezierLoaderSize
+import io.channel.bezier.compose.component.loader.properties.BezierLoaderVariant
+import io.channel.bezier.compose.foundation.ShadowStyle
+import io.channel.bezier.compose.foundation.bezierShadow
+import io.channel.bezier.extension.thenIf
+import io.channel.bezier.extension.toEmojiPainter
+import io.channel.bezier.icon.ArrowRight
+import io.channel.bezier.icon.Plus
+
+@Composable
+fun BezierFloatingButton(
+    text: String,
+    size: BezierFloatingButtonSize,
+    variant: BezierFloatingButtonVariant,
+    color: BezierFloatingButtonColor,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    prefixContent: BezierFloatingButtonContent? = null,
+    suffixContent: BezierFloatingButtonContent? = null,
+    isLoading: Boolean = false,
+    enabled: Boolean = true,
+) {
+    val colorSchemes = color.getColorSchemes(variant)
+    val backgroundColor = colorSchemes.backgroundColor().color
+    val contentColor = colorSchemes.contentColor().color
+
+    val shape = RoundedCornerShape(100.dp)
```

Comment:
애매한 100보다 그냥 원형으로 어떠신가요?
```suggestion
    val shape = CircleShape
```

Reply:
요건 피그마 정의상 100이어가지구 100으로 했었어요